src/microbe_stage/MicrobeHUD.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Looks like a removed Utf8 BOM

Reply:
I wonder why the format script didn't catch this change...

Reply:
Actually more of the other way around. It's a change from without to with BOM, at least that's what VSCode says which I'm pretty sure is correct.